pandas/tseries/base.py

```diff
@@ -17,6 +17,26 @@
 import pandas.algos as _algos
 
 
+
+class DatelikeOps(object):
+    """ common ops for DatetimeIndex/PeriodIndex, but not TimedeltaIndex """
+
+    def strftime(self, date_format):
+        """
+        Return a list of formmatted strings specified by date_format
+
```

Comment:
Can you add here a link somewhere to https://docs.python.org/2/library/datetime.html#strftime-and-strptime-behavior for an overview of the format specification?